.env.production

```
@@ -0,0 +1 @@
+NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_test_Q2N3ljTzzBgxO4uOlsv5BS7u00f3eocyfC
```

```suggestion
NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_live_4e9OH54b1NzmfES809cLedMu00F8m8gRka
```
Not sure which of the two it should be, just commit the auggestion wher you see fit :)